acceptance-tests/profiler-stress/runner.exe.sources

```diff
@@ -0,0 +1,156 @@
+../../external/Newtonsoft.Json/Src/Newtonsoft.Json/Bson/BsonBinaryType.cs
```

Comment:
`System.Net.Http.Formatting.dll` and various other assemblies in `mcs/` do it like this, and I can't find any makefiles that build and install `Newtonsoft.Json.dll` in the GAC.